Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/CriticalLogsAnalyticsHandler.cs

```diff
@@ -72,8 +77,11 @@ public void LogException(Exception exception, ReportData reportData, Object cont
                 { "type", "exception" },
                 { "category", reportData.Category },
                 { "scene_hash", reportData.SceneShortInfo.Name },
-                { "message", exception.Message },
+                { "message", IsPayloadSizeValid(exception.Message) ? exception.Message : "message is too long" },
```

Comment:
Maybe we can make "message is too long" a const since it's repeated in several places?